.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+    "presets": [
+        [
+            "next/babel",
+            {
+                "preset-env": {
+                    "debug": false,
+                    "targets": {
+                        "browsers": ">1%, not ie 11, not op_mini all"
+                    }
```

Comment:
а в next 12 уже swc по умолчанию
```
Compilation using the Next.js Compiler is 17x faster than Babel and enabled by default since Next.js version 12.
```
https://nextjs.org/docs/advanced-features/compiler#why-swc

надо через browserslist
https://nextjs.org/docs/advanced-features/customizing-postcss-config#customizing-target-browsers